Uplift/Assets.xcassets/back-arrow.imageset/Contents.json

```diff
@@ -2,15 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
+      "filename" : " Dark.png",
```

Comment:
lowercase and remove space

Reply:
` Dark.png` is just the name of the file when it was downloaded, in xcassets it's named appropriately.